supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc

```diff
@@ -0,0 +1,245 @@
+
+*Use it*: yes
+
+*Incorrect forms*: 
+
+*See also*: xref:consumer-rhds[consumer]
+
+[discrete]
+[[ldap]]
+==== image:images/yes.png[yes] LDAP (noun)
+*Description*: The Lightweight Directory Access Protocol (LDAP) defines an industry standard for accessing and maintaining directory servers, such as Red Hat Directory Server. By default, the LDAP protocol is unencrypted. Do not expand the abbreviation on the first usage.
+
+*Use it*: yes
+
+*Incorrect forms*: Lightweight Directory Access Protocol
+
+*See also*: xref:ldaps[LDAPS], xref:starttls[STARTTLS]
+
+[discrete]
+[[ldaps]]
+==== image:images/yes.png[yes] LDAPS (noun)
+*Description*: The Lightweight Directory Access Protocol over Secure Socket Layer (LDAPS) uses the TLS protocol to encrypt LDAP traffic. Do not expand the abbreviation on the first usage.
+
+*Use it*: yes
+
+*Incorrect forms*: Lightweight Directory Access Protocol over Secure Socket Layer
+
+*See also*: xref:ldap[LDAP], xref:starttls[STARTTLS]
+
+[discrete]
+[[master]]
+==== image:images/no.png[no] master (noun)
+*Description*: Do not use "master" to refer to a supplier.
+
+*Use it*: no
+
+*Incorrect forms*: 
+
+*See also*: xref:supplier[supplier]
+
+[discrete]
+[[object-class]]
+==== image:images/yes.png[yes] object class (noun)
+*Description*: Object classes in an entry control which attributes are optional and which are required. Write as two words when you refer to object classes in general.
+
+*Use it*: yes
+
+*Incorrect forms*: objectClass
+
+*See also*: xref:objectclass[objectClass]
+
+[discrete]
+[[objectclass]]
+==== image:images/yes.png[yes] objectClass (noun)
+*Description*: The objectClass attribute in an LDAP entry stores the object classes of this entry. 
+
+*Use it*: yes
+
+*Incorrect forms*: object class, objectclass
+
+*See also*: xref:object-class[object class]
+
+[discrete]
+[[red-hat-directory-server]]
+==== image:images/yes.png[yes] Red Hat Directory Server (noun)
+*Description*: Red Hat Directory Server (RHDS) is an LDAPv3-compliant directory server and the name of the product. Use the full product name in titles of guides. Outside of titles, refer to the product as "Directory Server". Use the product name without an article. Do not use the acronym "RHDS" in documentation.
+
+*Use it*: yes
+
+*Incorrect forms*: RHDS
+
+*See also*: xref:directory-server[Directory Server]
+
+[discrete]
+[[replica]]
+==== image:images/yes.png[yes] replica (noun)
+*Description*: A replica is a copy of the Directory Server database on a different host. For example, a consumer can also be called a replica because it has a copy of the data received from the supplier.
+
+*Use it*: yes
+
+*Incorrect forms*: 
+
+*See also*: 
+
+[discrete]
+[[slave]]
+==== image:images/no.png[no] slave (noun)
+*Description*: Do not use "slave" to refer to a consumer or hub.
+
+*Use it*: no
+
+*Incorrect forms*: 
+
+*See also*: xref:consumer-rhds[consumer], xref:hub[hub]
+
+[discrete]
+[[ssl]]
```

Comment:
Looks like there is another "ssl" ID already too sorry, can you also fix this to be ssl-rhds or something else unique?

Reply:
Fixed